### DISCUSSION TITLE
How customize coversions folder ?

### DISCUSSION THREAD

**Discussion by hooman-mirghasemi** [spatie/laravel-medialibrary#2644] [Q&A]
How can I customize conversion folder, I'd prefer all images of my users in big size in a folder and all small size in an other folder.
like:
conversion(folder)
-----user-avatar(folder)
---------- big(folder)
-----------------abc.jpg
-----------------efg.png
-----------small(folder)
-----------------abc.jpg
-----------------efg.png

and dont add -samll.jpg or -larage.jpg at end of images.

**Comment by erikn69** [comment]
Why do everyone always open a issue or discussion for things that are explained in the documentation?
[Here ](https://spatie.be/docs/laravel-medialibrary/v9/advanced-usage/using-a-custom-directory-structure#breadcrumb) and [here](https://spatie.be/docs/laravel-medialibrary/v9/advanced-usage/naming-files#naming-original-and-conversion-files)

**Reply by erikn69** [reply]
>I need custom folder not all images in conversions folder.
I need have small folder to save all small size. and medium folder not

@hooman-mirghasemi still the same answer, I do the same, and i only use the information from that documentation, use `name-of-the-conversion` to differentiate what size it is, you can customize the paths and do what you need

**Comment by hooman-mirghasemi** [comment]
I need custom folder not all images in conversions folder.
I need have small folder to save all small size. and medium folder not
